spec/integration/guest_view_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+describe 'the guest view', type: :feature do
+  #include Capybara::DSL
+
+  describe 'the home page' do
+    xit "has a navbar" do
+      visit root_url
+      within("#nav-bar") do
+        expect(page).to have_link("Main")
+        expect(page).to have_link("Menu")
+        expect(page).to have_link("Login")
+      end
+    end
+
+    xit "goes to the menu page" do
+      page.click_link("Menu")
+      expect(current_path).to eq(menu_path)
+    end
+  end
+
+  describe 'the menu view' do
+    xit "shows all the menu items" do
+      visit(menu_path)
+      expect(page).to have_content("some cheese stuff")
+      expect(page).to have_content("milk")
+    end
+
+    xit "has an inner navbar for menu categories" do
+      visit(menu_path)
+      within("#nav-bar2") do
+        Category.all.each do |category|
+          expect(page).to have_link(category.name)
+        end
+      end
+    end
+
+    xit "links to the correct menu categories" do
+      visit(menu_path)
+      page.click_link("Main meals")
+      expect(current_path).to eq(menu_path(category))
+    end
+  end
+
+  describe 'the category menu view' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.